methods_and_recursion/bottles_of_beer.rb

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env ruby
+
+def bottles(beer)
+  return 0 if beer == -1
```

Comment:
I think this would read better on line 13, i.e. `bottles(beer - 1) if beer >= 0`